src/redux/books/Books.js

```diff
@@ -1,9 +1,35 @@
 import React, { useState } from 'react';
 import { v4 as uuidv4 } from 'uuid';
-import BookList from '../components/BookList';
-import AddBook from '../components/AddBook';
+import BookList from '../../components/BookList';
+import AddBook from '../../components/AddBook';
 
-const Books = () => {
+const ADD = 'ADD';
+const REMOVE = 'REMOVE';
```

Comment:
- [ ] With reference to the _duck pattern_, assigning values to _ADD_BOOK_ and _REMOVE_BOOK_ action types should be something like:
```ruby
       const ADD_BOOK = bookstore/books/ADD_BOOK;
```

and

```ruby
      const REMOVE_BOOK = bookstore/books/REMOVE_BOOK;
```

Kindly ensure to improve upon these aspects.